src/BsReactstrap__ButtonDropdown.re

```diff
@@ -1,20 +1,12 @@
-[@bs.module "reactstrap"] external buttonDropdown : ReasonReact.reactClass = "ButtonDropdown";
+[@bs.module "reactstrap"]
+external buttonDropdown : ReasonReact.reactClass = "ButtonDropdown";
 
 [@bs.obj]
-external makeProps : (
+external makeProps : (~isOpen: bool=?, ~toggle: 'a=?, unit) => _ = "";
```

Comment:
can you annotate `toggle`, please?
I guess it's smth like
`ReactEventRe.Form.t => unit`
or
`unit => unit`

I don't know if it accepts any arguments

Reply:
Seems like it passes the event right?

from reactstrap:
```
toggle(e) {
    if (this.props.disabled) {
      return e && e.preventDefault();
    }

    return this.props.toggle(e);
```